src/main/java/com/netgrif/quartz/mongodb/MongoDBJobStore.java

```diff
@@ -1,40 +1,32 @@
-/*
- * $Id: MongoDBJobStore.java 253170 2014-01-06 02:28:03Z waded $
- * --------------------------------------------------------------------------------------
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- *
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-package com.novemberain.quartz.mongodb;
+package com.netgrif.quartz.mongodb;
 
 import com.mongodb.MongoCommandException;
 import com.mongodb.MongoException;
 import com.mongodb.client.MongoClient;
 import com.mongodb.client.MongoCollection;
 import com.mongodb.client.MongoDatabase;
-import com.novemberain.quartz.mongodb.db.MongoConnector;
-import com.novemberain.quartz.mongodb.util.Keys;
+import com.netgrif.quartz.mongodb.db.MongoConnector;
+import com.netgrif.quartz.mongodb.util.Keys;
 import org.bson.Document;
 import org.quartz.*;
-import org.quartz.Calendar;
 import org.quartz.Trigger.CompletedExecutionInstruction;
 import org.quartz.Trigger.TriggerState;
 import org.quartz.impl.matchers.GroupMatcher;
 import org.quartz.spi.*;
+import org.quartz.Calendar;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import java.io.BufferedInputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
 import java.util.*;
 import java.util.Map.Entry;
 import java.util.concurrent.TimeUnit;
 
 public class MongoDBJobStore implements JobStore, Constants {
 
+    private static final Logger log = LoggerFactory.getLogger(MongoDBJobStore.class);
```

Comment:
Can we use ``lombok`` instead of this? Or we don't want to use lombok dependency?